tests/v2/validation/provisioning/hosted/aks/hosted_provisioning_test.go

```diff
@@ -134,19 +117,33 @@ func (h *HostedAKSClusterProvisioningTestSuite) testProvisioningHostedAKSCluster
 	require.NoError(h.T(), err)
 	assert.NotEmpty(h.T(), clusterToken)
 
+	err = nodestat.IsNodeReady(rancherClient, clusterResp.ID)
+	require.NoError(h.T(), err)
+
+	podResults, podErrors := pods.StatusPods(rancherClient, clusterResp.ID)
+	assert.NotEmpty(h.T(), podResults)
+	assert.Empty(h.T(), podErrors)
+
 	return clusterResp, nil
 }
 
-func (h *HostedAKSClusterProvisioningTestSuite) testScalingAKSNodePools(rancherClient *rancher.Client, clusterName string) {
+func (h *HostedAKSClusterProvisioningTestSuite) testScalingAKSNodePools(rancherClient *rancher.Client, clusterName string, cluster *management.Cluster) {
 	cloudCredential, err := azure.CreateAzureCloudCredentials(rancherClient)
 	require.NoError(h.T(), err)
 
-	cluster, err := h.testProvisioningHostedAKSCluster(rancherClient, clusterName, cloudCredential)
-	require.NoError(h.T(), err)
+	if cluster == nil {
```

Comment:
Would this work when running both Standard and Admin? I think it's current form, it will only provision the admin cluster. I didn't see any hosted Jenkins runs, so I can't confirm this.

Reply:
It should, I thought I included Jenkins runs but I'll double-check.